src/extras/module-types.js

```diff
@@ -33,6 +33,9 @@
       if (cssContentType.test(contentType))
         return res.text()
         .then(function (source) {
+          source = source.replace(/url\(\s*?(["']??)((?:\\.|[^\s:"'])+?)\1\s*?\)/g, function (match, quotes, id) {
+            return 'url(' + quotes + systemJSPrototype.resolve(id, url) + quotes + ')';
```

Comment:
This shouldn't use the SystemJS resolver, instead it should use a normal path resolver only without the custom SystemJS hooks (CSS imports don't use import maps for example). You should be able to import a lower-level resolution function from the resolve module here.